README.md

```diff
@@ -4,6 +4,25 @@ This is a boilerplate application which can be forked to start a new project for
 
 ## Prerequisite
 
+### With the docker image builder
+
+The app-builder docker image [from this repository](https://github.com/LedgerHQ/ledger-app-builder) contains all needed tools and library to build and load an application.
+You can download it from the ghcr.io docker repository:
+
+```shell
+docker pull ghcr.io/ledgerhq/ledger-app-builder/ledger-app-builder-full
+```
+
+You can then enter this development environment by executing the following command from the directory of the application `git` repository:
+
+```shell
+docker run --rm -ti  -v "$(realpath .):/app" ghcr.io/ledgerhq/ledger-app-builder/ledger-app-builder-full
```

Comment:
Maybe adding `--user "$(id -u)":"$(id -g)"` can be a good idea.
It helps when docker needs to be run under `sudo`.

Reply:
Indeed!